addon/components/md-loader.js

```diff
@@ -19,12 +16,23 @@ export default Component.extend(UsesSettings, {
 
   classNameBindings: ['isBarType:progress:preloader-wrapper', 'active:active', 'size'],
 
-  mode: oneWay('_mdSettings.loaderMode'),
+  mode: null,
   percent: 0,
-  size: oneWay('_mdSettings.loaderSize'),
+  size: null,
```

Comment:
you don't really need these, since they're `undefined` if not specified at all. I kind of like to do what you've done though, since it's a clue to developers that this is part of the API surface